### ISSUE TITLE
Suggest to be able to make several tests inside an if call

### ISSUE THREAD

**Issue by Spheerys** [splittingred/If#9]
Instead of imbricate IF, it could be usefull to be able to make several tests inside an if call.

Something like this : 
[[!If?
   &subject=`[[+name]]`
   &operator=`EQ`
   &operand=`George`
   &if2=`OR`
   &subject2=`[[+name]]`
   &operator2=`EQ`
   &operand2=`Bob` 
   &if3=`OR`
   &subject2=`[[+name]]`
   &operator2=`EQ`
   &operand2=`John` 
   &then=`Hey my friends !`
   &else=`Sorry I don't know you. Go away.`
]]

I'm not sure is the best way to call the parameter, but it's OMHO the only think it's missing to this usefull addon :)

**Comment by jpdevries** [comment]
I think this is your best best

```
[[!If?
&subject=`[[+name]]`
&operator=`inarray`
&operand=`George,Bob,John`
&then=`Hey my friends!`
&else=`Sorry I don't know you. Go away.`]]
```

**Comment by Spheerys** [comment]
Yes your solution is more clean, but less flexible in case of different tests :)

**Comment by jpdevries** [comment]
True, I think in those cases a simple snippet to perform your conditional may be more practical

**Comment by Spheerys** [comment]
Do you have an example ? :)

**Comment by jpdevries** [comment]
``` php
<?php
$s = '';
switch($input.toLowerCase()) {
  case 'george':
  case 'bob':
  case 'john':
  $s .= 'Hey my friends!';
  break;

  default:
  $s .= "Sorry I don't know you. Go away.";
  break;
}
return $s;
```

**Comment by Spheerys** [comment]
Thank you ! :)
